ipc/provider/src/manager/subnet.rs

```diff
@@ -111,6 +112,20 @@ pub trait SubnetManager: Send + Sync + TopDownCheckpointQuery {
     /// Returning as a `String` because the maximum value for an EVM
     /// networks is a `U256` that wouldn't fit in an integer type.
     async fn get_chain_id(&self) -> Result<String>;
+
+    /// Gets the genesis information required to bootstrap a child subnet
+    async fn get_genesis_info(&self, subnet: &SubnetID) -> Result<SubnetGenesisInfo>;
+}
+
+#[derive(Debug)]
+pub struct SubnetGenesisInfo {
+    pub bottom_up_checkpoint_period: u64,
+    pub msg_fee: TokenAmount,
+    pub majority_percentage: u8,
+    pub active_validators_limit: u16,
+    pub min_collateral: TokenAmount,
+    pub genesis_epoch: ChainEpoch,
+    pub validators: Vec<subnet_actor_getter_facet::Validator>,
```

Comment:
we should use our own definition of `Validator`. Using `subnet_actor_getter_facet::Validator` will be carried to other functions or contract calls or in the future other network types. It might haunt us in the futures.